SixTrack/sixtrack.s

```diff
@@ -25492,6 +25495,20 @@ C     Convert r(1), r(2) from U(0,1) -> rvec0 as Gaussian with cutoff mcut (#sig
           call postpr(91-i,nnuml)
 +ei
    70   continue
++ei
++if stf
+        do 70 i=1,ndafi
++if .not.cr
+          call postpr(i)
++ei
++if cr
+          write(93,*) 'Calling POSTPR nnuml=',nnuml
+          endfile (93,iostat=ierro)
+          backspace (93,iostat=ierro)
+          call postpr(i,nnuml)
++ei
+  70      continue
++ei
         call sumpos
```

Comment:
It is actually compiled into sixve.f but the final code is 

    if(ipos.eq.1.and.napx.eq.0) then
      open(10,file='fort.10',form='formatted',status='unknown')
        do 70 i=1,ndafi
          call postpr(91-i)
    70   continue
        call sumpos
        goto 520
      endif

but napx shouldn't be zero since it represents number of particles tracked,which means this code will never run and this is a bug?